src/utils/listDisplay.ts

```diff
@@ -0,0 +1,6 @@
+export function getListDisplay(entity: object, listDisplay: string[]) {
+  if (!listDisplay) {
+    return []
+  }
```

Comment:
This whole function isn't useful given above comments, but in addition:

- The typing of the arguments don't say that listdisplay can be null (and that's right, it can't), so this condition is wrong
- For safety you could throw an exception instead